### PR TITLE
Renamed the solution

### PR DESCRIPTION
**Pull request by Halpin994**
Renamed the solution from "MovieDB" to "myWebsite" as I am building the
website using the project I introduced myself to C# MVC with.

I cleared out almost all of the MovieDB related code.

I have embedded my LinkedIn profile on my index page.

### CODE REVIEW

**Comment by sean-halpin** [comment]
Remember to modify the path to the .sln that Jenkins & MSBuild point to after you merge. 
Also IIS might need to be repointed.

File: packages/repositories.config [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <repositories>
   <repository path="..\MovieDB\packages.config" />
```
Comment (sean-halpin) [inline packages/repositories.config]: Can probably remove this line ?

File: myWebsite/bin/myWebsite.dll.config [hunk 2]
```diff
@@ -9,10 +9,12 @@
...
   </configSections>
   <connectionStrings>
+    <!--
     <add name="DefaultConnection" connectionString="Data Source=(LocalDb)\v11.0;AttachDbFilename=|DataDirectory|\aspnet-MovieDB-20160715041019.mdf;Initial Catalog=aspnet-MovieDB-20160715041019;Integrated Security=True" providerName="System.Data.SqlClient" />
     <add name="MoviesDBEntities" connectionString="metadata=res://*/Models.MoviesDBModel.csdl|res://*/Models.MoviesDBModel.ssdl|res://*/Models.MoviesDBModel.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=(LocalDB)\v11.0;attachdbfilename=|DataDirectory|\MoviesDB.mdf;integrated security=True;MultipleActiveResultSets=True;App=EntityFramework&quot;" providerName="System.Data.EntityClient" />
     <add name="Model1" connectionString="data source=(LocalDB)\v11.0;attachdbfilename=|DataDirectory|\MoviesDB.mdf;integrated security=True;multipleactiveresultsets=True;application name=EntityFramework" providerName="System.Data.SqlClient" />
```
Comment (softdev87) [inline myWebsite/bin/myWebsite.dll.config]: Remove the lines MoviesDBEntities and Model1

File: myWebsite/Web.config [hunk 3]
```diff
@@ -9,10 +9,12 @@
     <section name="entityFramework" type="System.Data.Entity.Internal.ConfigFile.EntityFrameworkSection, EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
   </configSections>
   <connectionStrings>
+    <!--
     <add name="DefaultConnection" connectionString="Data Source=(LocalDb)\v11.0;AttachDbFilename=|DataDirectory|\aspnet-MovieDB-20160715041019.mdf;Initial Catalog=aspnet-MovieDB-20160715041019;Integrated Security=True" providerName="System.Data.SqlClient" />
     <add name="MoviesDBEntities" connectionString="metadata=res://*/Models.MoviesDBModel.csdl|res://*/Models.MoviesDBModel.ssdl|res://*/Models.MoviesDBModel.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=(LocalDB)\v11.0;attachdbfilename=|DataDirectory|\MoviesDB.mdf;integrated security=True;MultipleActiveResultSets=True;App=EntityFramework&quot;" providerName="System.Data.EntityClient" />
```
Comment (sean-halpin) [inline myWebsite/Web.config]: Remove the lines for MoviesDBEntities & Model1

File: myWebsite/Models/MoviesDBModel.Context.cs [hunk 4]
```diff
@@ -7,7 +7,7 @@
 // </auto-generated>
 //------------------------------------------------------------------------------
 
-namespace MovieDB.Models
+namespace myWebsite.Models
```
Comment (sean-halpin) [inline myWebsite/Models/MoviesDBModel.Context.cs]: Remove all the files under myWebsite/Models/ relating to Movies

**Comment by sean-halpin** [comment]
Remove the file MoviesDB.mdf. 
Also Type movies into the solution explorer and try delete all of it. 
Delete the controllers and views you aren't using anymore. 
Make sure it builds and push your changes. Then I'll approve the PR.

**Review by sean-halpin — APPROVED** [review]
Looks Good, Merge the PR and check Jenkins can build & IIS can host.

**Pull request merged**